cfgov/unprocessed/css/pages/home.less

```diff
@@ -17,20 +17,25 @@
 }
 
 .o-well__home {
-    .respond-to-min( @bp-sm-min, {
-        .content-l__large-gutters {
+    .respond-to-min( 1100px, {
```

Comment:
I noticed that the spacing between the equal height cards was not being preserved on the home page when the screen size reduced towards 1024px:
<img width="1006" alt="screen shot 2017-05-02 at 9 27 52 pm" src="https://cloud.githubusercontent.com/assets/778171/25651839/f244a382-2f99-11e7-96d6-fa007fc80147.png">

So I added a custom 1100px breakpoint for the cards on the home page.